heutagogy/views.py

```diff
@@ -161,6 +161,8 @@ def post(self, id):
                 bookmark.read = aniso8601.parse_datetime(update['read'])
             else:
                 bookmark.read = None
+        if 'tags' in update:
+            bookmark.tags = update['tags']
```

Comment:
Is it possible to refactor the code to something like this?
```
new_bookmark = {
  url: update['url'] ? urldefrag(update['url']).url : bookmark.url,
  title: update['title'] || bookmark.title,
  timestamp: update['timestamp'] ? aniso8601.parse_datetime(update['timestamp']) : bookmark.timestamp,
  read: update['read'] ? aniso8601.parse_datetime(update['read']) : bookmark.read
  tags: update['tags'] || bookmark.tags
}

Reply:
I think not. `bookmark` is not a plain map but rather an object returned from SQLAlchemy, which contains some internal field for object tracking and SQLAlchemy needs.